packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -1558,6 +1632,23 @@ export class Domain extends DomainBase implements IDomain, ec2.IConnectable {
       }
     }
 
+    let offPeakWindowOptions: OffPeakWindowOptions = {
+      enabled: props.offPeakWindowOptions?.enabled ?? true,
```

Comment:
> All domains created before this date have the off-peak window disabled by default. 

So we can't just default to true here, as that will represent a behavior change to existing domains. Rather, without setting `offPeakWindow` (i.e. `offPeakWindow: undefined`) what we have to do is document the service-level behavior (enabled post 2/16/23, disabled prior). Then, we just pass `undefined` into the L1 and let the service handle the default.